api/tests/src/search/backend/test_load_opportunities_to_index.py

```diff
@@ -1,12 +1,52 @@
+from unittest.mock import Mock
+
 import pytest
 
+from src.constants.lookup_constants import OpportunityStatus
+from src.db.models.opportunity_models import (
+    CurrentOpportunitySummary,
+    Opportunity,
+    OpportunitySearchIndexQueue,
+    OpportunitySummary,
+)
 from src.search.backend.load_opportunities_to_index import (
     LoadOpportunitiesToIndex,
     LoadOpportunitiesToIndexConfig,
 )
 from src.util.datetime_util import get_now_us_eastern_datetime
 from tests.conftest import BaseTestClass
-from tests.src.db.models.factories import OpportunityFactory
+from tests.src.db.models.factories import OpportunityFactory, OpportunitySearchIndexQueueFactory
+
+
+@pytest.fixture
+def mock_search_client():
+    client = Mock()
+    # Mock the alias exists check
+    client.alias_exists.return_value = True
+    # Mock empty initial index
+    client.scroll.return_value = [Mock(records=[])]
+    return client
+
+
+@pytest.fixture
+def test_opportunity():
```

Comment:
Why not use the Opportunity factory for this?

Reply:
Good call, this was simpler to use the factory. At first I was thinking of a shared opportunity for many tests, but the factory is easier.